src/main/common/typeconversion.h

```diff
@@ -16,6 +16,11 @@
  */
 #pragma once
 
+#define i2i8(i) ((int8_t)((i) % 0xff))
+#define i2i16(i) ((int16_t)((i) % 0xffff))
+#define i2u8(i) ((uint8_t)((i) & 0xff))
+#define i2u16(i) ((uint16_t)((i) & 0xffff))
```

Comment:
I agree - only left them lower to match existing - and if they are ever converted to actual functions...